spring-data-neo4j/src/main/java/org/springframework/data/neo4j/repository/config/Neo4jMappingContextFactoryBean.java

```diff
@@ -27,12 +27,35 @@
  *
  * @author Mark Angrish
  * @author Nicolas Mervaillie
+ * @author Michael J. Simons
  */
 public class Neo4jMappingContextFactoryBean extends AbstractFactoryBean<Neo4jMappingContext> implements
 		ApplicationContextAware {
 
+	private final String sessionFactoryBeanName;
+
 	private ListableBeanFactory beanFactory;
 
+	/**
+	 * Uses the default session factory name {@link Neo4jRepositoryConfigurationExtension#DEFAULT_SESSION_FACTORY_BEAN_NAME}
+	 * for finding the session factory passed to the {@link Neo4jMappingContext}.
+	 *
+	 * @deprecated
+	 */
+	public Neo4jMappingContextFactoryBean() {
```

Comment:
Missing `@Deprecated` on constructor level.

Reply:
I don't understand. It's there, so it should be missing?